src/prog_algs/predictors/monte_carlo.py

```diff
@@ -36,6 +38,7 @@ class MonteCarlo(Predictor):
     default_parameters = { 
         'n_samples': 100  # Default number of samples to use, if none specified
     }
+    PredictionResults = namedtuple('PredictionResults', ["times", "inputs", "states", "outputs", "event_states", "time_of_event"])
```

Comment:
This should probably be defined in the same file as Prediction so it doesn't have to be defined multiple times. Then you can just import it (i.e., `from .prediction import PredictionResults`